test/e2e/landing-page.spec.ts

```diff
@@ -1,11 +1,13 @@
 import { test, expect } from '@playwright/test';
+import { deleteAllCertificates } from '../../app/models/certificate.server';
 import { loggedInAsUser } from './utils';
 
 test.describe('Landing Page', () => {
   loggedInAsUser();
 
   test.beforeEach(async ({ page }) => {
-    await page.goto('/');
+    await deleteAllCertificates();
+    return page.goto('/');
```

Comment:
Same question as before, why do we need to call `deleteAllCertificates` here?

Reply:
I added a fake certificate in `certificate-available.spec`. I noticed that this landing page test state is somehow affected from the certificate from other test. So one of the button text is different which leads this landing page test to a failure.

Reply:
I see that you are deleting the certificates in that spec after the tests are run. Could you confirm that this is needed?